node/index.js

```diff
@@ -96,250 +100,252 @@ app.post('/api/create_link_token', function (request, response, next) {
+        {
+          name: 'Harry Potter',
+          iban: 'GB33BUKB20201555555555',
+          address: {
+            street: ['4 Privet Drive'],
+            city: 'Little Whinging',
+            postal_code: '11111',
+            country: 'GB',
+          },
+        },
+      );
+      const recipientId = createRecipientResponse.recipient_id;
 
-        client.createPayment(
-          recipientId,
-          'payment_ref',
-          {
+      const createPaymentResponse = await client.paymentInitiationPaymentCreate(
+        {
+          recipient_id: recipientId,
+          reference: 'payment_ref',
+          amount: {
             value: 12.34,
             currency: 'GBP',
           },
-          function (error, createPaymentResponse) {
-            prettyPrintResponse(createPaymentResponse);
-            const paymentId = createPaymentResponse.payment_id;
-            PAYMENT_ID = paymentId;
-            const configs = {
-              user: {
-                // This should correspond to a unique id for the current user.
-                client_user_id: 'user-id',
-              },
-              client_name: 'Plaid Quickstart',
-              products: PLAID_PRODUCTS,
-              country_codes: PLAID_COUNTRY_CODES,
-              language: 'en',
-              payment_initiation: {
-                payment_id: paymentId,
-              },
-            };
-            if (PLAID_REDIRECT_URI !== '') {
-              configs.redirect_uri = PLAID_REDIRECT_URI;
-            }
-            client.createLinkToken(
-              {
-                user: {
-                  // This should correspond to a unique id for the current user.
-                  client_user_id: 'user-id',
-                },
-                client_name: 'Plaid Quickstart',
-                products: PLAID_PRODUCTS,
-                country_codes: PLAID_COUNTRY_CODES,
-                language: 'en',
-                redirect_uri: PLAID_REDIRECT_URI,
-                payment_initiation: {
-                  payment_id: paymentId,
-                },
-              },
-              function (error, createTokenResponse) {
-                if (error != null) {
-                  prettyPrintResponse(error);
-                  return response.json({
-                    error,
-                  });
-                }
-                response.json(createTokenResponse);
-              },
-            );
-          },
-        );
-      },
-    );
+        },
+      );
+      PAYMENT_ID = createPaymentResponse.payment_id;
+
+      const configs = {
+        user: {
+          // This should correspond to a unique id for the current user.
+          client_user_id: 'user-id',
+        },
+        client_name: 'Plaid Quickstart',
+        products: PLAID_PRODUCTS,
+        country_codes: PLAID_COUNTRY_CODES,
+        language: 'en',
+        payment_initiation: {
+          payment_id: PAYMENT_ID,
+        },
+      };
+      if (PLAID_REDIRECT_URI !== '') {
```

Comment:
completely optional and for your consideration as this isn't necessary a better pattern (more concise though):

```node
configs = {
...
  redirect_uri = PLAID_REDIRECT_URI ? PLAID_REDIRECT_URI : undefined
...
}
```

benefits: 
* you can make `configs` immutable
* no conditional inserts after `configs`

drawbacks: 
* ternaries detract from clarity imo
* may or may not cause issues storing `undefined` directly into an `object` :'). we will see.

Reply:
this pattern also shows up throughout the code, so whatever your decision propagate it please!

Reply:
I think I am going to leave it cause I don't want any issues with storing an undefined property...